bench/real_construction_bench.cpp

```diff
@@ -0,0 +1,66 @@
+#include <benchmark/benchmark.h>
+#include <benchmark_helpers.hpp>
+
+/** benchmarks the construction of a real number
+ *  for some varying parameter
+ *  (number of tree nodes, or number of digits)
+ */ 
+
+const int MIN_TREE_NODES = 10; 
+const int MAX_TREE_NODES = 10000;
+const int MULTIPLIER_TC = 10;  // for range evaluation of tree construction benchmarks
+/// benchmarks the construction speed of doing a op= b, n times, where n is the set of powers of
+/// MULTIPLIER_TC between MIN_TREE_NODES and MAX_TREE_NODES
+void BM_RealOperationTreeConstruction(benchmark::State& state, boost::real::OPERATION op) {
+    for (auto i : state) {
+    boost::real::real a ("1234567891");
+    boost::real::real b ("9876532198");
+
+    // We keep the precision constant here because in constructing the *= tree, we would get way more digits
+    // than in +=, -= trees. This should make the benchmarks more meaningful
+    a.set_maximum_precision(10);
+
+        for (int i = 0; i < state.range(0); i++)
```

Comment:
Please add braces even that they are not necessary to keep code style consistent